src/nvim/ex_getln.c

```diff
@@ -394,6 +394,7 @@ static uint8_t *command_line_enter(int firstc, long count, int indent)
     try_enter(&tstate);
     apply_autocmds(EVENT_CMDLINELEAVE, (char_u *)firstcbuf, (char_u *)firstcbuf,
                    false, curbuf);
+    apply_autocmds(EVENT_MODELEAVE, (char_u *)"cmdline", NULL, false, curbuf);
```

Comment:
This event didn't trigger, on `autocmd ModeLeave cmdline echom 'Hello'` or on even `autocmd ModeLeave * echom`, but the command `autocmd CmdlineLeave * echom` executes, I even tried by changing this line to `apply_autocmds(EVENT_MODELEAVE, (char_u *)firstcbuf, (char_u *)firstcbuf, false, curbuf);`, but it didn't work

Reply:
But rest of Events triggered like `autocmd ModeLeave insert echom 'Hello'`  works fine